api/src/main/resources/workbench-api.yaml

```diff
@@ -4426,7 +4426,10 @@ definitions:
         description: The RAS base host name.
       accessRenewalLookback:
         type: number
-        description: the point when we give users the option to update their compliance items.
+        description: the point when we give users the option to update the affirmations needed to use the system.
+      complianceTrainingRenewalLookback:
```

Comment:
This duplicates the existing field at line 4376

Reply:
Thanks for pointing this out. I fixed the duplication.